modules/shared/src/main/scala/org/tessellation/currency/dataApplication/package.scala

```diff
@@ -202,6 +213,12 @@ object BaseDataApplicationContextualOps {
       ): F[DataApplicationValidationErrorOr[Unit]] =
         service.validateFee(gsOrdinal)(update.asInstanceOf[Signed[D]])
 
+      override def estimateFee(gsOrdinal: SnapshotOrdinal)(update: DataUpdate)(
+        implicit context: Context,
+        A: Applicative[F]
+      ): F[EstimatedFee] =
+        service.estimateFee(gsOrdinal)(update.asInstanceOf[Signed[D]])
```

Comment:
why cast this to a `Signed`?

Reply:
The class is defined in terms of the type variable `D <: DataUpdate`
`update` is of type `Signed[DataUpdate]` and the method requires a `Signed[D]`

Reply:
As discussed in our meeting, the method does NOT want a `Signed[D]`. I'll fix it.